data/campaigns/Son_Of_The_Black_Eye/scenarios/08_Silent_Forest.cfg

```diff
@@ -226,7 +226,7 @@
 
         [message]
             speaker="Kapou'e"
-            message= _ "Men, capture one of these elves and interrogate him. I am sure that will get us some answers."
+            message= _ "Men, capture one of these elves for interrogation. I am sure that will get us some answers."
```

Comment:
While I think it's reasonable for Kapou'e to disregard the fact that not all elvish combatants are male, this works for me.